### PR TITLE
添加用户atcoder和codeforces的信息项

### PR DESCRIPTION
**Pull request by hexisyztem**
使用户可以展示自己的codeforces和atcoder账号

### CODE REVIEW

File: user/attachinfo/constant.py [hunk 1]
```diff
@@ -15,3 +15,6 @@
 
 # The max length limitation of user gravatar
 MAX_GRAVATAR_LENGTH = 128
+
+# The max length limitation of username
+MAX_USERNAME_LENGTH = 32
```
Comment (LaPluses) [inline user/attachinfo/constant.py]: MAX_USERNAME_LENGTH make me confusing, could you rename this constant?

File: user/attachinfo/models.py [hunk 2]
```diff
@@ -10,3 +10,5 @@ class AttachInfo(models.Model):
     location = models.CharField(max_length=MAX_LOCATION_LENGTH, blank=True)
     about = models.CharField(max_length=MAX_ABOUT_LENGTH, blank=True, default=DEFAULT_ABOUT)
     gravatar = models.CharField(max_length=MAX_GRAVATAR_LENGTH, blank=True)
+    codeforces = models.CharField(max_length=MAX_USERNAME_LENGTH, blank=True, default='errorerror')
```
Comment (LaPluses) [inline user/attachinfo/models.py]: Default is 'errorerror'? 
IMO, leave blank may be a good choice.

**Comment by hexisyztem** [comment]
Just a joke.http://codeforces.com/profile/errorerror <http://codeforces.com/profile/errorerror>

File: user/attachinfo/models.py (hunk 2 above)
Comment (hexisyztem) [reply user/attachinfo/models.py]: just a kidding........ http://codeforces.com/profile/errorerror

**Comment by hexisyztem** [comment]
添加用户的学号信息项, 并预留关于用户的'性别'信息项

**Comment by hexisyztem** [comment]
okay

**Pull request merged**